src/app/build.gradle

```diff
@@ -73,7 +73,7 @@ docker {
 // creates folder at app/demo with up to date orderly test data
 task generateTestData() {
     doLast {
-        String image = "vimc/orderly:master"
+        String image = "vimc/orderly:vimc-2929"
```

Comment:
more branch pins (can this not come from the config/orderly_server_version, in some future maintenance PR?)

Reply:
yeah this whole test setup is a mess. why is this a gradle task, that calls shell commands, that it itself utlimately called by a shell script?